ui/src/org.ts

```diff
@@ -458,3 +489,26 @@ export const openAnchorProjectModal = async (
     gnosisSafeAddress,
   });
 };
+
+async function getClaimedIdentity(
+  contract: ClaimsContract,
+  address: string
+): Promise<identity.RemoteIdentity | undefined> {
+  const radicleIdBytes = await contract.getClaimed(address);
+  if (!radicleIdBytes) {
+    return undefined;
+  }
+  const urn = identitySha1Urn(radicleIdBytes);
+  let identity;
+  try {
+    identity = await proxy.client.remoteIdentityGet(urn);
+  } catch {
+    return undefined;
```

Comment:
we should probably check for a 404 here and re-throw all other errors.

Reply:
Fixed